### PR TITLE
add __Dymola_selection implementation

### LINKED ISSUE #748: Add __Dymola_selection option to AixLib export
## What is the problem?
- Currently, TEASER exports always store all variables by default, one can use the Dymola output options to select to store only outputs, but this is not really useful as not all relevant data from the exported models are stored in outputs.
- Due to this result files might get really big especially for buildings with a lot of thermal zones.

## Why do we want to solve it?

- More granular selection of stored data in result files and reduction of file size for fast post-processing 

## How do we want to solve it?
Use the __Dymola_selection option. 
Therefore I implemented an additional parameter `export_vars` in the `export_aixlib()` method. The parameter is a dictionary that specifies which parameters should be stored. Following you can find an example and the results of this. I also added some documentation in the example `e2_export_aixlib_models`. 

### Test with the following export selected:
```    export_vars = {
        "HeatingDemands": ["*multizone.PHeater*", "*multizone.PHeatAHU"],
        "CoolingDemands": ["*multizone.PCooler*", "*multizone.PCoolAHU"],
        "Temperatures": ["*multizone.TAir*", "*multizone.TRad*"]
    }
```

Standard: **8166 KB**
Reduced result file size: **429 KB**

Here one can see how the results are displayed:
image

If not providing the parameter `export_vars` to the `export_aixlib()` method, everything behaves like "normal".

### PR DESCRIPTION
**Pull request by DaJansenGit**
closes #748. Relevant information can be found in the issue!

### CODE REVIEW

File: teaser/project.py [hunk 1]
```diff
@@ -1059,6 +1060,11 @@ def export_aixlib(
         report: boolean
             if True a model report in form of a html and csv file will be
             created for the exported project.
+        export_vars: dict[str:list]
+            dict where key is a name for this variable selection and value is a
+             list of variables to export, wildcards can be used, multiple
+             variable selections are possible.
```
Comment (FWuellhorst) [inline teaser/project.py:1066]: Why the whitespace in the second and third line?

Comment (FWuellhorst) [reply teaser/project.py:1066]: Link to claytex o.s.? 
Also, explicitly note that this will only work for Dymola?

**Review by FWuellhorst — APPROVED** [review]
Minor comments on docs, nice work

File: teaser/project.py (hunk 1 above)
Comment (DaJansenGit) [reply teaser/project.py]: you are right! both solved with 7cd15517

**Review by FWuellhorst — APPROVED** [review]

**Pull request merged**